src/main/react-front-end/src/views/Login/CreateAccountComponent.js

```diff
@@ -197,7 +189,7 @@ export default class CreateAccountComponent extends Component {
 
           <ValidatorForm
             ref="email"
-            onSubmit={this.onEmailNextClicked}>
```

Comment:
We need to check if the email is already present in the database before registering incase the user tries to create an account with an already existing email address. Please revert this change.